snap/snapcraft.yaml

```diff
@@ -21,6 +21,11 @@ environment:
   ZOO_LOG_DIR: ${SNAP_COMMON}/var/log/zookeeper
   ZOOCFGDIR: ${SNAP_DATA}/etc/zookeeper
 
+lint:
+  ignore:
+    - library:
+      - usr/lib/jvm/**/lib*.so
```

Comment:
**question:** What was this regarding? What issue were you seeing?

Reply:
Not necessary, but during `snapcraft pack` a dozen of annoying unused library warnings show up in the linter:

Lint warnings:                                                                                                                                                                         
- library: libattach.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libattach.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libawt_headless.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libawt_headless.so'. (https://snapcraft.io/docs/linters-library)                                  
- library: libdt_socket.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libdt_socket.so'. (https://snapcraft.io/docs/linters-library)                                        
- library: libextnet.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libextnet.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libfontmanager.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libfontmanager.so'. (https://snapcraft.io/docs/linters-library)                                    
- library: libinstrument.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libinstrument.so'. (https://snapcraft.io/docs/linters-library)                                      
- library: libj2gss.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libj2gss.so'. (https://snapcraft.io/docs/linters-library)                                                
- library: libj2pcsc.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libj2pcsc.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libj2pkcs11.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libj2pkcs11.so'. (https://snapcraft.io/docs/linters-library)                                          
- library: libjaas.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjaas.so'. (https://snapcraft.io/docs/linters-library)                                                  
- library: libjavajpeg.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjavajpeg.so'. (https://snapcraft.io/docs/linters-library)                                          
- library: libjdwp.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjdwp.so'. (https://snapcraft.io/docs/linters-library)                                                  
- library: libjimage.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjimage.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libjsig.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjsig.so'. (https://snapcraft.io/docs/linters-library)                                                  
- library: libjsound.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libjsound.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: liblcms.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/liblcms.so'. (https://snapcraft.io/docs/linters-library)                                                  
- library: libmanagement.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libmanagement.so'. (https://snapcraft.io/docs/linters-library)                                      
- library: libmanagement_agent.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libmanagement_agent.so'. (https://snapcraft.io/docs/linters-library)                          
- library: libmanagement_ext.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libmanagement_ext.so'. (https://snapcraft.io/docs/linters-library)                              
- library: libmlib_image.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libmlib_image.so'. (https://snapcraft.io/docs/linters-library)                                      
- library: libnet.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libnet.so'. (https://snapcraft.io/docs/linters-library)                                                    
- library: libnio.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libnio.so'. (https://snapcraft.io/docs/linters-library)                                                    
- library: libprefs.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libprefs.so'. (https://snapcraft.io/docs/linters-library)                                                
- library: librmi.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/librmi.so'. (https://snapcraft.io/docs/linters-library)                                                    
- library: libsaproc.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libsaproc.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libsctp.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libsctp.so'. (https://snapcraft.io/docs/linters-library)                                                  
- library: libsunec.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libsunec.so'. (https://snapcraft.io/docs/linters-library)                                                
- library: libunpack.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libunpack.so'. (https://snapcraft.io/docs/linters-library)                                              
- library: libzip.so: unused library 'usr/lib/jvm/java-11-openjdk-amd64/lib/libzip.so'. (https://snapcraft.io/docs/linters-library)

Reply:
Ah gotcha, yeah no problem to keep then, thanks :)